specs/reverse_sentence_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require_relative '../lib/reverse_sentence'
+
+describe "reverse sentence" do
+  describe "basic tests" do
+    describe "test 1" do
+      it "reverse a sentence with two words" do
+        test_string = "hello, world"
+
+        reverse_sentence(test_string)
+
+        test_string.must_equal "world hello,"
+      end
+    end
+
+    describe "test 2" do
+      it "reverse a sentence with three words" do
+        test_string = "Yoda is awesome!"
+
+        reverse_sentence(test_string)
+
+        test_string.must_equal "awesome! is Yoda"
+      end
+    end
+  end
+
+  # if it's a string parameter, check for empty
+  describe "empty string input test" do
+    it "reverse an empty sentence" do
```

Comment:
I think this empty string test and the nil test below it could be moved into the edge cases section, and a few of those tests (such as sentence with multiple words) could be moved into the basic tests section.